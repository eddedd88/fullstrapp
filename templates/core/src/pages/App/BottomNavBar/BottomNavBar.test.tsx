import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import BottomNavBarWithRouter, { BottomNavBar } from './BottomNavBar'

describe('BottomNavBar', () => {
  it('renders correctly', () => {
    const renderer = ShallowRenderer.createRenderer()
    const tree = renderer.render(
      <BottomNavBar
        classes={{
          root: 'test'
        }}
        location={{
          pathname: '/test'
        }}
      />
    )
    expect(tree).toMatchSnapshot()
  })

  it('renders with router correctly', () => {
    const renderer = ShallowRenderer.createRenderer()
    const tree = renderer.render(<BottomNavBarWithRouter />)
    expect(tree).toMatchSnapshot()
  })
})
