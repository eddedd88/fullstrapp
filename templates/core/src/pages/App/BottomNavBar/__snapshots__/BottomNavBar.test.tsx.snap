// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNavBar renders correctly 1`] = `
<WithStyles(BottomNavigation)
  classes={
    Object {
      "root": "test",
    }
  }
  showLabels={true}
  value="/test"
>
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon={<pure(LocationCityIcon) />}
    label="Feed"
    value="/feed"
  />
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon={<pure(GridOnIcon) />}
    label="Grid"
    value="/grid"
  />
  <WithStyles(BottomNavigationAction)
    component={[Function]}
    icon={<pure(PersonIcon) />}
    label="Profile"
    value="/profile"
  />
</WithStyles(BottomNavigation)>
`;

exports[`BottomNavBar renders with router correctly 1`] = `
<Route>
  [Function]
</Route>
`;
